tests/test_integration/test_frame_extraction.py

```diff
@@ -1,51 +1,490 @@
+import datetime
+import json
+import os
+import re
 from pathlib import Path
+from typing import Optional
 
 import pytest
+import typer
+from typer.testing import CliRunner
 
 from crabs.bboxes_labelling.extract_frames_to_label_w_sleap import (
     get_list_of_sleap_videos,
 )
 
 
-@pytest.fixture(autouse=True, scope="class")
-def input_video_dir():
-    return Path(__file__).parents[1] / "data" / "clips"
+@pytest.fixture()
+def input_data_dir() -> str:
+    return str(Path(__file__).parents[1] / "data" / "clips")
```

Comment:
What's the benefit of a fixture (which takes three lines to define and one extra line in the argument to pytest) over a simple file-scope global?
```python
INPUT_DATA_DIR = str(Path(__file__).parents[1] / "data" / "clips")
```

Reply:
Hmm [the internet says this](https://stackoverflow.com/questions/75863152/pytest-fixture-vs-global-variable). 🤔

Reply:
But if you promise not to change the global, perhaps we're OK?

Reply:
I have this doubt so many times...

Reply:
but yeah happy to make it a global in this case